daemon/Libraries/Symlink/Environment/Windows.cs

```diff
@@ -63,7 +59,23 @@ public Windows(Symlink parent)
         /// <returns></returns>
         public bool Create(string symlink, string absolutePath)
         {
-            return CreateSymbolicLink(symlink, absolutePath, _parent.GetAbsolutePathType(absolutePath));
+            var processOptions = new ProcessOptions()
+            {
+                Executable = "cmd",
+                Arguments = (_parent.GetAbsolutePathType(absolutePath) == Symlink.SymbolicLink.Directory)
```

Comment:
Does this actually work as a normal user, or do you have to be admin?

Reply:
This works as a normal user, I tested this manually in a shell.